Autoload.php

```diff
@@ -121,10 +121,36 @@ public function findFile($class)
                 return $dir . DIRECTORY_SEPARATOR . $classFile;
             }
         }
+
+        if ($resolvedPath = $this->findComposerMagentoModuleFile($class)) {
+            return $resolvedPath;
+        }
         
         return false;
     }
 
+    /**
+     * Allow the autoloader to find composer magento module files in controllers dir
+     * These files don't follow the same class name/location conventions
+     *
+     * @param $class
+     * @return bool|string
+     */
+    private function findComposerMagentoModuleFile($class){
```

Comment:
How about renaming this `findMagentoControllerFile`? This functionality is specific to controllers and will work for any Magento module, not only 'Composered' ones?